### PR TITLE
Remove configuration remote update

### PR DESCRIPTION
**Pull request by aethernet**
Remove the check for an updated configuration from the server.

Change-type: patch

### CODE REVIEW

**Comment by aethernet** [comment]
I self-certify!

File: lib/gui/etcher.ts [hunk 1]
```diff
@@ -206,32 +205,6 @@ async function createMainWindow() {
...
-				const autoUpdaterConfig: AutoUpdaterConfig = onlineConfig?.autoUpdates
-					?.autoUpdaterConfig ?? {
-					autoDownload: false,
-				};
-				for (const [key, value] of Object.entries(autoUpdaterConfig)) {
-					autoUpdater[key as keyof AutoUpdaterConfig] = value;
-				}
-				const checkForUpdatesTimer =
-					onlineConfig?.autoUpdates?.checkForUpdatesTimer ?? 300000;
-				checkForUpdates(checkForUpdatesTimer);
```
Comment (lurch) [inline lib/gui/etcher.ts:229]: Looks like this also removes the call to `checkForUpdates` - was that intentional?

Comment (aethernet) [reply lib/gui/etcher.ts:229]: aouch! No not intentional. Good catch!

Comment (aethernet) [reply lib/gui/etcher.ts:229]: fixed in https://github.com/balena-io/etcher/pull/4006

**Pull request merged**